docs/features/software-templates/ui-options-examples.md

```diff
@@ -441,3 +441,30 @@ repoUrl:
 `secretsKey` is the key used within the template secrets context to store the credential and `additionalScopes` is any additional permission scopes to request.
 
 The supported `additionalScopes` values are `gerrit`, `github`, `gitlab`, `bitbucket`, and `azure`.
+
+## RepoBranchPicker
+
+The input props that can be specified under `ui:options` for the `RepoBranchPicker` field extension.
+
```

Comment:
Based on the discussion in the Scaffolder SIG just now let's just add a note here about the fact that autocomplete only works for Bitbucket. Perhaps like this:

```Markdown
:::note
The autocomplete feature of this picker currently only supports Bitbucket
:::
```

Reply:
Is this something you'd expect in `ui-options-examples.md` or writing-templates.md`? Or both?

Reply:
Why do we have this duplicated? Sorry, missed that. Would make sense to have it in one place and then link

Reply:
`ui-options-examples.md` contains more information about the input props, while `writing-templates.md` is more general. It's the same for `RepoUrlPicker`.

Reply:
Ok, going to leave this as is then but feel like an opportunity to simplify things in the future 👍